platforms/hyperledger-besu/configuration/README.md

```diff
@@ -1,49 +1,49 @@
+Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments (from the sample file found [here](./samples/network-besu.yaml)).
 
 ### Step 2
-Ideally, the configuration should be run by the following command from the root of the project. The [platforms/shared/configuration/site.yaml](../../shared/configuration/site.yaml) is the master playbook which does basic environment setup, kubernetes environment setup and then calls platform specific deployment playbooks.
+Execute the playbook by running the command below - executed from the root of the project:
 ```
-ansible-playbook platforms/shared/configuration/site.yaml -e "@./platforms/samples/network-besu.yaml"
+ansible-playbook platforms/shared/configuration/site.yaml -e "@/path/to/network-besu.yaml"
 ```
-Ensure that you have added an **SSH Key** with read-write permission to your git repository before executing the playbook. Follow steps in [README](../../shared/configuration/README.md).<br>
-If you just want to run this deploy-network.yaml playbook, then, from the *platforms/Hyperledger Besu/configuration* directory, run the following command for deploying the network
+The [platforms/shared/configuration/site.yaml](../../shared/configuration/site.yaml) is the master playbook which does basic environment setup, configures the Kubernetes cluster and then calls platform specific deployment playbooks.
+
+You can also only run the platform specific deployment playbooks by running the command below - executed from the root of the project:
 ```
-ansible-playbook deploy-network.yaml -e "@./samples/network-besu.yaml"
+ansible-playbook platforms/hyperledger-besu/deploy-network.yaml -e "@/path/to/network-besu.yaml"
 ```
-(Above command assumes that network-besu.yaml is present in samples directory).<br>
-The playbook will create resources in the Kubernetes cluster(s) and will intermittently wait for resources to be created before proceeding. You may want to check the cluster(s) for any errors.
-<br>
 
 ### Step 3
-After deploy-network.yaml has run successfully, your nodes or the participants in the Hyperledger Besu network should be up and running.
+After your Ansible command has completed. your nodes or the participants in the Hyperledger Besu network should be up and running. We are working on a verification document for Hyperledger Besu.
 
 ## Miscellaneous
 
-1. openssl.conf: This is the configuration file used to generate the Root CA certificates for Hyperledger Besu-CA.
+1. `./openssl.conf`: This is the configuration file used to generate the Root CA certificates for Besu-CA.
+
+2. If you want to reset the network, i.e. delete all created resources while setting up the Hyperledger Besu network, then run the following command from the root folder of the project:
+    ```
+    ansible-playbook platforms/hyperledger-besu/cleanup.yaml -e "@/path/to/network-besu.yaml"
```

Comment:
is it cleanup or reset-network? use site.yaml -e reset=true as well

Reply:
For Besu specifically the file is called `cleanup.yaml`. I'll add the `-e "reset=true"` option as well, for all platforms.

Reply:
Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)